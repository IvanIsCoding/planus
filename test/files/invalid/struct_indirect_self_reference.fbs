struct Foo {
  x: uint64;
  y: Bar;
}

struct Bar {
  x: uint64;
  y: Foo;
}