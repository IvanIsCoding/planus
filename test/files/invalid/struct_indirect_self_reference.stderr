[0m[1m[38;5;9merror[0m[1m: Structs are not allowed to contain themselves, as it implies infinite size[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/struct_indirect_self_reference.fbs:1:1
  [0m[34mâ”‚[0m
[0m[34m1[0m [0m[34mâ”‚[0m struct Foo {
  [0m[34mâ”‚[0m [0m[34m----------[0m [0m[34mFoo contains itself through Bar[0m

[0m[1m[38;5;9merror[0m[1m: Structs are not allowed to contain themselves, as it implies infinite size[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/struct_indirect_self_reference.fbs:6:1
  [0m[34mâ”‚[0m
[0m[34m6[0m [0m[34mâ”‚[0m struct Bar {
  [0m[34mâ”‚[0m [0m[34m----------[0m [0m[34mBar contains itself through Foo[0m

Error: Bailing because of errors
